voice-assistant/m5stack-atom-echo.yaml

```diff
@@ -96,22 +108,21 @@ binary_sensor:
     disabled_by_default: true
     entity_category: diagnostic
     id: echo_button
-    on_multi_click:
-      - timing:
-          - ON FOR AT LEAST 350ms
-        then:
-          - voice_assistant.start:
-      - timing:
-          - ON FOR AT LEAST 350ms
-          - OFF FOR AT LEAST 10ms
-        then:
-          - light.turn_on:
-              id: led
-              blue: 100%
-              red: 0%
-              green: 0%
-              effect: pulse
-          - voice_assistant.stop:
+    on_click:
+      - if:
+          condition: voice_assistant.is_running
+          then:
+            - light.turn_off: led
+            - voice_assistant.stop:
+          else:
+            - light.turn_on:
+                id: led
+                blue: 100%
+                red: 0%
+                green: 0%
+                brightness: 50%
+                effect: none
+            - voice_assistant.start_continuous:
```

Comment:
Let’s just make it do 1 conversation push to start talking. Not start a conversation

Reply:
Ok